integration-test/cli.Tests.ps1

```diff
@@ -52,10 +50,10 @@ Describe 'Console apps (<framework>) - native AOT publish' -ForEach @(
         Remove-Item "./console-app/bin/Release/$framework/publish" -Recurse -ErrorAction SilentlyContinue
     }
 
-    It "uploads symbols and sources" -Skip:($IsMacOS -and $framework -eq 'net7.0') {
+    It "uploads symbols and sources" {
         $result = RunDotnetWithSentryCLI 'publish' 'console-app' $True $True $framework
         $result.ScriptOutput | Should -AnyElementMatch "Preparing upload to Sentry for project 'console-app'"
-        if ($IsWindows -or ($IsLinux -and $framework -eq 'net7.0'))
+        if ($IsWindows)
         {
             $extension = $IsLinux ? '' : '.pdb'
```

Comment:
I guess the confidtional is redundant now eh (inside this block it's always windows so always a pdb)?

Reply:
right, I've missed the linux bit, thanks